ui/components/app/home-notification/home-notification.stories.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import HomeNotification from './home-notification.component';
+
+export default {
+  title: 'UI/HomeNotification',
+  id: __filename,
+  component: HomeNotification,
+  parameters: {},
+  argTypes: {
+    checkboxText: {
+      control: 'text',
+    },
+    acceptText: {
+      control: 'text',
+    },
+    ignoreText: {
+      control: 'text',
+    },
+    descriptionText: {
+      control: 'text',
+    },
+    infoText: {
+      control: 'text',
+    },
+  },
+};
+
+const noop = () => {
+  // noop
+};
+
+export const DefaultStory = (args) => (
+  <HomeNotification {...args} onAccept={noop} onIgnore={noop} />
+);
+
+DefaultStory.storyName = 'Default';
+
+DefaultStory.args = {
+  ignoreText: 'foo',
```

Comment:
It's nice to show the UI with any props that usually would be used.
```suggestion
    acceptText: 'Accept text',
  checkboxText: 'Checkbox text',
  checkboxTooltipText: 'Checkbox tooltip text',
  ignoreText: 'Ignore text',
  infoText: 'Info text',
  descriptionText: 'Description text',
```